package.json

```diff
@@ -111,7 +112,7 @@
     "watch:build:umd": "cross-env FILE=umd SOURCEMAPS=true NODE_ENV=development rollup -c -w",
     "watch:build:cjs": "cross-env FILE=cjs SOURCEMAPS=true NODE_ENV=development rollup -c -w",
     "test": "mocha -r esm --recursive",
-    "test:build": "cross-env TEST_BUILD=true mocha",
+    "test:build": "cross-env TEST_BUILD=true mocha -r esm",
```

Comment:
This should not include `-r esm`.  The point of this test is to test the built bundle as consumed by node.  There should be no ES modules in the tested bundle.